docs/dev_guide/diagnostic_ref/dpct1018.rst

```diff
@@ -44,4 +44,46 @@ parameter and you observe performance issues, consider changing the values of
 
 If the ``incx`` parameter of the ``cublasSetVector`` equals the ``incy`` parameter,
 but is greater than 1 and you observe performance issues, consider changing the
-values of ``incx`` and ``incy``.
\ No newline at end of file
+values of ``incx`` and ``incy``.
+
+For example, this original CUDA\* code:
+
+.. code-block:: cpp
+   :linenos:
+
+   void foo() {
+     const int element_num = 128;
+     const int h_inc = 128;
+     const int d_inc = 128;
+     cublasSetVector(element_num, sizeof(float), data, h_inc, d_data, d_inc);
+   }
+
+results in the following migrated SYCL\* code:
+
+.. code-block:: cpp
+   :linenos:
+
+   void foo() {
+     const int element_num = 128;
+     const int h_inc = 128;
+     const int d_inc = 128;
+     /*
+     DPCT1018:0: The cublasSetVector was migrated, but due to parameter h_inc
+     equals to parameter d_inc but greater than 1, the generated code performance
+     may be sub-optimal.
+     */
+     dpct::matrix_mem_copy((void *)d_data, (void *)data, d_inc, h_inc, 1,
+                           element_num, sizeof(float));
+   }
+
+which is rewritten to:
+
+.. code-block:: cpp
+   :linenos:
+
+   void foo() {
+     const int element_num = 128;
+     const int h_inc = 1;
+     const int d_inc = 1;
+     dpct::get_default_queue().memcpy(d_data, data, sizeof(float) * element_num).wait();

```

Comment:
@zhiweij1 , please add comment to explain why Line 88 works.

Reply:
because the `h_inc' and `d_inc` has been changed from 128 to 1. So there is no padding between each elements.

Reply:
Added.